src/features/Applet/Schedule/Calendar/YearView/YearView.tsx

```diff
@@ -1,6 +1,46 @@
-import { DateLocalizer } from 'react-big-calendar';
+import { DateLocalizer, NavigateAction } from 'react-big-calendar';
+import { startOf, add } from 'react-big-calendar/lib/utils/dates';
 
-export const YearView = () => <div>Year View</div>;
+import { MonthCalendar } from './MonthCalendar';
+import { StyledYear } from './YearView.styles';
+import { YearViewProps } from './YearView.types';
+
+export const YearView = ({ date, localizer, events, onNavigate, components }: YearViewProps) => {
+  const months = [];
+  const firstMonth = startOf(date, 'year');
+  const { setDate, setActiveView } = components;
+
+  for (let i = 0; i < 12; i++) {
```

Comment:
you could do like 'const months = Array.from({ length: 12}, (_, i) => (<MonthCalendar
    key={i}....'

Reply:
done